### PR TITLE
CICD MVP (rough draft)

### PR DESCRIPTION
**Pull request by christopherkeim**
# PR Description

This contains base level resources for a minimal viable CICD pipeline for continuous deployment of our website to GitHub Pages.

- Minimal changes to JS source code for configuration with GitHub Pages 
- Addition of `deploy-to-github-pages.yml` GitHub Action; no `Linting`, `Test` steps within the Action
- Addition of `setup.sh` for local developer environment setup 
- Addition of `test.sh` for local dev server testing

## Further Work

We still need to:

- Add `Linting` to CICD pipeline
- Add `Test` to CICD pipeline
- Build out more robust local testing automation

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/user-story.md
Comment (kensonjohnson) [inline .github/ISSUE_TEMPLATE/user-story.md:1]: Not sure how these templates work, but nice to have

File: README.md [hunk 1]
```diff
@@ -2,10 +2,69 @@
 
 Template from [here](https://github.com/tailwindtoolbox/Landing-Page/blob/master/index.html).
 
-## Usage
+## Requirements
 
-**Assumes you have Node installed. You can get it for your machine [here](https://nodejs.org/en/download)**
+For our developer setup we have assumed Ubuntu 22.04 as our base OS.
```
Comment (kensonjohnson) [inline README.md:7]: We will not be making this assumption. 90% of developers use Windows or Mac as their daily driver.

**Review by kensonjohnson — APPROVED** [review]
Looks good! :shipit:

File: .github/ISSUE_TEMPLATE/user-story.md
Comment (christopherkeim) [reply .github/ISSUE_TEMPLATE/user-story.md:1]: Whenever you make a new issue, you can choose to "use a template" and this template is just a User Story that you can fill in. Forces you to think socially!

**Pull request merged**